pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -4439,6 +4792,30 @@ class WorldSessionActor extends Actor with MDCContextAware {
             log.info(s"GenericObject: $player is MAX with an unexpected weapon - ${definition.Name}")
         }
       }
+      else if(action == 36) { //Looking For Squad ON
```

Comment:
magic numbers. can we get an enum pls?

Reply:
I'd like `PlanetsideAttributeMessage` to make sense myself!